pkg/ext/stores/tokens/tokens.go

```diff
@@ -564,7 +569,10 @@ func (t *Store) Update(
 		return nil, false, apierrors.NewNotFound(GVR.GroupResource(), oldToken.Name)
 	}
 
-	sessionID := t.auth.SessionID(ctx)
+	sessionID, err := t.auth.SessionID(ctx)
+	if err != nil {
+		return nil, false, apierrors.NewInternalError(err)
```

Comment:
Maybe something along the lines of
```suggestion
		return nil, false, apierrors.NewInternalError(fmt.Errorf("error getting the authentication token: %w", err))
```